README.md

```diff
@@ -181,7 +181,11 @@ curl -X PUT --header 'Content-Type: application/json' --header 'Accept: applicat
   ],
   "topicConfig": {},
   "partitions": 1,
-  "replicationFactor": 1
+  "replicationFactor": 1,
+  "alertConfigList": [
+  ],
+  "connectorConfigList": [
+  ]
```

Comment:
Not sure this pattern is right.

Otherwise stream-metadata becomes a monolith.

Wouldn't it make more sense to have a `/sources` and `/sinks` be first class entities ? Perhaps with an FK to to a stream?

This way "stream metadata stays concise and manageable"